shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -375,6 +378,59 @@ public void systemInsetHandlesFullscreenNavbarLeft() {
     assertEquals(100, viewportMetricsCaptor.getValue().paddingRight);
   }
 
+  @Test
+  @TargetApi(30)
+  @Config(sdk = 30)
+  public void systemInsetGetInsetsFullscreen() {
+    RuntimeEnvironment.setQualifiers("+land");
+    FlutterView flutterView = spy(new FlutterView(RuntimeEnvironment.systemContext));
+    ShadowDisplay display =
```

Comment:
Thanks for writing the test. These shadows are a bit of a pain to figure out the APIs for.